lib/status-bar-manager.coffee

```diff
@@ -14,9 +19,10 @@ class StatusBarManager
 
   initialize: (@statusBar) ->
 
-  update: (currentMode) ->
+  update: (currentMode, currentSubmode) ->
+    currentFullMode = currentMode + "$" + currentSubmode if currentSubmode?
```

Comment:
Thank you.
I've committed a fix in 91069db61fd16d6f7c6a03880cceea0fc8eeb235 so the code doesn't rely on order, keeping the ability to match only on the `mode`. If we don't keep `currentFullMode` we'd have to enumerate all submodes in ContentsByMode.
Which alternative would you prefer?